tests/gcloud-testing-core/src/main/java/com/google/cloud/runtime/jetty/test/ContainerRunner.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.runtime.jetty.test;
+
+import com.google.cloud.runtime.jetty.test.annotation.Local;
+import com.google.cloud.runtime.jetty.test.annotation.Remote;
+
+import org.junit.Ignore;
+import org.junit.runner.Description;
+import org.junit.runner.notification.RunNotifier;
+import org.junit.runners.BlockJUnit4ClassRunner;
+import org.junit.runners.model.FrameworkMethod;
+import org.junit.runners.model.InitializationError;
+
+public class ContainerRunner extends BlockJUnit4ClassRunner {
+  private boolean localTestsEnabled = false;
+  private boolean remoteTestsEnabled = false;
```

Comment:
There are only two states (local|remote). So, a single boolean should be enough to represent it. Finding a name for it can be tricky though. :)